css/css-grid/abspos/grid-positioned-item-dynamic-change-006.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang=en class="reftest-wait">
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Grid positioned item dynamic change</title>
+<link rel="author" title="Filipp Riabchun" href="mailto:talpa@yandex.ru">
+<link rel="help" href="https://drafts.csswg.org/css-grid-1/#abspos" title="9. Absolute Positioning">
+<meta name="assert" content="This test checks that positioned items react to grid track size changes.">
```

Comment:
You need to add a reference file and link it from here with: `<link rel="match" href="..." />`.

Reply:
What's the recommended way to position items in the reference files?

Reply:
Whatever is simpler for you, but something different than the actual test.

For example it could be a similar grid with an abspos element, but the first item is already 50px height. Or it could not use grid at all, that would be fine too.